models/migrations/v16.go

```diff
@@ -13,8 +13,8 @@ import (
 	"github.com/go-xorm/xorm"
 )
 
-// RepoUnit describes all units of a repository
-type RepoUnit struct {
+// V16RepoUnit describes all units of a repository
+type V16RepoUnit struct {
```

Comment:
Better is to move type inside migration function so that you don't need to rename it and don't need table name function